ncm-sudo/src/main/perl/sudo.pm

```diff
@@ -197,8 +199,11 @@ sub generate_privilege_lines {
         my $ln = $info{PRIVILEGE_USER()}->getValue;
         $ln .= "\t" . $info{PRIVILEGE_HOST()}->getValue;
         $ln .= "= (". $info{PRIVILEGE_RUNAS()}->getValue . ")\t";
-        $ln .= $info{PRIVILEGE_OPTS()}->getValue . "\t"
-        if defined $info{PRIVILEGE_OPTS()};
+	if (defined($info{PRIVILEGE_OPTS()})) {
+		$ln .= $info{PRIVILEGE_OPTS()}->getValue;
+		$ln .= ':' if $ln !~ /:$/;
```

Comment:
just to understand this, the ending colon is mandatory, https://github.com/ajf8/configuration-modules-core/commit/e79f799ab4bfa0025b869c7bf7e124be0b048f7f tried to force that via schema, the new code forces it via the component?